app/Jobs/TileSingleImage.php

```diff
@@ -43,14 +57,17 @@ class TileSingleImage extends Job implements ShouldQueue
     /**
      * Create a new job instance.
      *
-     * @param Image $image The image to generate tiles for.
+     * @param Image $file The image to generate tiles for.
      *
      * @return void
      */
-    public function __construct(Image $image)
+    public function __construct(Image $file)
```

Comment:
This class is not really more generalized if you still restrict it to `Image` here. Maybe you want a `Biigle\FileCache\Contracts\File` instead?

Reply:
resolved with 4f42cb7bf35ace46d1cbeacf4f6b98fc776ec3f0